cli/chaosli/cmd/context.go

```diff
@@ -0,0 +1,388 @@
+
+	fmt.Println("\nLets take a look at the status of your Targeted Nodes...")
+
+	for i := 0; i < len(targetsInfo); i++ {
+		node := targetsInfo[i]
+		info := "\t🥸  Node Name: " + node.Name + "\n" +
+			"\t📲 Node Addresses: \n"
+
+		for _, address := range node.Status.Addresses {
+			info += "\t\t⭕️ Type: " + string(address.Type) + "\n" +
+				"\t\tℹ️  Address: " + address.Address + "\n"
+		}
+
+		info += "\t📜 Node Conditions:\n"
+
+		for _, status := range node.Status.Conditions {
+			info = info +
+				"\t\t⭕️ Type: " + string(status.Type) + "\n" +
+				"\t\tℹ️  Status: " + string(status.Status) + "\n" +
+				"\t\t🤨 Reason: " + status.Reason + "\n\n"
+			percentCollectCondition[string(status.Type)+"/"+string(status.Status)] += 1.0 / float64(len(targetsInfo))
+		}
+
+		if i < MAXTARGETSHOW {
+			fmt.Printf(info + "\n\n")
+		}
+	}
+
+	PrintSeparator()
+
+	percentInfo := "Lets look at the overall status...\n"
+
+	for key, value := range percentCollectCondition {
+		roundedValue := value * 100.00
+		percentInfo += "\tCondition Type/Status: " + key + "\n" +
+			"\tPercent:               " + fmt.Sprint(math.Round(roundedValue*100)/100) + "%\n\n"
+	}
+
+	fmt.Println(percentInfo)
+	PrintSeparator()
+}
+
+func checkKubectl() error {
+	cmd := exec.Command("kubectl", "get", "pods", "-n", "chaos-engineering")
+	_, err := cmd.Output()
+
+	return err
+}
+
+var contextCmd = &cobra.Command{
+	Use:   "context",
+	Short: "contextualizes disruption config",
+	Long:  `makes use of kubectl to give a better idea of how big the scope of the disruption will be.`,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		path, _ := cmd.Flags().GetString("path")
+		return validatePath(path)
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		path, _ := cmd.Flags().GetString("path")
+		contextualize(path)
+	},
+}
+
+func contextualize(path string) {
+	disruption := ReadUnmarshalValidate(path)
+
+	err := checkKubectl()
+	if err != nil {
+		log.Fatalf("Could not find/use kubectl command, make sure it is in your PATH variable and that all authorizations for the command are set (login to your authorization provider (e.g. Your VPN).")
+	}
+
+	targets, err := contextTargetsSize(disruption)
+	if err != nil {
+		log.Fatalf("Could not grab context regarding size and names of targets: %v", err)
+	}
+
+	podsData, nodesData, err := grabDataForTargets(targets, disruption)
+	if err != nil {
+		log.Fatalf("Attempted to grab data for targets and failed: %v", err)
+	}
+
+	if disruption.Spec.Level == types.DisruptionLevelPod {
+		printPodStatus(podsData)
+		printContainerStatus(podsData)
+	} else {
+		printNodeStatus(nodesData)
+	}
+}
+
+func init() {
+	contextCmd.Flags().String("path", "", "The path to the disruption file to be contextualized.")
+	contextCmd.Println("This flag requires that you are connected to a cluster and kubectl works as intended on that cluster. All the results of this flag will be based on your current kubectx.")
```

Comment:
```suggestion
	contextCmd.Println("This command requires that you are connected to a cluster and kubectl works as intended on that cluster. All the results of this command will be based on your current kubectx.")
```

This is referring to `chaosli context`, yes? I think "command" is a better noun to use then

Reply:
yeah that sounds a lot better.